services/s3control/src/it/java/software.amazon.awssdk.services.s3control/S3ControlIntegrationTest.java

```diff
@@ -20,6 +20,7 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
+import org.assertj.core.api.ThrowableAssert;
```

Comment:
Looks like this dependency wasn't used?

Reply:
Missed that, built with the quick flag...